src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Ball {
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    private String color;
+    private int number;
```

Comment:
Access modifiers are missing for the fields 'color' and 'number'. It's a good practice to explicitly define the access level for class members. If you intend these to be private, please add the 'private' modifier.